### DISCUSSION TITLE
Over fitting neural network

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#720] [Q&A]
Hello everyone, I would like to ask, if you want to change the number of neurons and the number of network layers, which parameter of the diagram needs to be adjusted, thank you
![image](https://user-images.githubusercontent.com/73742098/121348562-136c3e80-c95b-11eb-98d9-85221dda6f61.png)

**Comment by marian-code** [comment]
The one in the fitting net category is what you want to adjust.

**Reply by lammps-vasp** [reply]
Thank you for your answer
What does the neuron in the descriptor represent

**Comment by njzjz** [comment]
There are two different NNs in the model so you will see two NN parameters.

**Reply by lammps-vasp** [reply]
thank you for your answer
Can we fit the energy separately? If only fitting the energy, which model should be used? If only adjusting the number of network layer and neurons, which model parameters should be adjusted

**Reply by njzjz** [reply]
I don't understand what you were meaning. The output of one NN is the input of another NN.

**Reply by lammps-vasp** [reply]
If I want to change the number of nerve layers and neurons, which parameter should I adjust, such as [25,50100] and [250,250,250], which one should I adjust, or both

Good luck

**Reply by njzjz** [reply]
You are free to adjust both. Here is an example: https://deepmd.readthedocs.io/en/latest/troubleshooting/howtoset_netsize.html

**Reply by lammps-vasp** [reply]
Thank you! I'll try

**Comment by lammps-vasp** [comment]
thank you for your answer
Can we fit the energy separately? If only fitting the energy, which model should be used? If only adjusting the number of network layer and neurons, which model parameters should be adjusted




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "deepmodeling/deepmd-kit"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月10日(星期四) 凌晨0:08
***@***.***&gt;;
抄送:&nbsp;"Zhifeng ***@***.******@***.***&gt;;
主题:&nbsp;Re: [deepmodeling/deepmd-kit] Over fitting neural network (#720)





 
There are two different NNs in the model so you will see two NN parameters.
